web/docs/tql2/functions/otherwise.md

```diff
@@ -0,0 +1,39 @@
+# otherwise
+
+Returns a `fallback` value if `primary` is `null`.
+
+```tql
+otherwise(primary:expr, fallback:expr)
+```
```

Comment:
With the other docs, we tried to go away from "expr" as the type. Not sure whether there is a better alternative here?

Reply:
I think this might be oneplace where `expr` truly fits. It can be an expression returning an arbitrary type with the caveat that currently fallback must also evaluate to the same type.

Reply:
I also thought about this, but the only alternatives I could think of would be

* No type at all. This would break the style, which not providing any benefit.
* Using a generic `T` in both cases (to signal that they must have the same type). I would be an entire 1-off though and personally I wouldnt want to introduce another concept like this into the docs.
* `any`, which I think we use somewhere/talked about, but may appear that both of these can be any unrelated type.

Additionally, I would kind of what to retain `any` for a future, *true* any type (essentially an alias for union<all_types...>)

Reply:
That the fallback must be the same type is also an artifact of not having the heterogeneous evaluator yet.

I think it's fine to be a bit imprecise here for now and use `otherwise(primary: any, fallback: any) -> any`. It's better than `expr` because `expr` just raises questions if it's not used anywhere else.